3457 - 0/triangle.rb

```diff
@@ -0,0 +1,27 @@
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+def triangle(aaa, bbb, ccc)
```

Comment:
I've already seen this solution, and it's fine, but please in future use more meaningful variable names, your code should be like a story :)